api/src/main/java/gov/va/vro/api/resources/ClaimMetricsResource.java

```diff
@@ -149,4 +149,37 @@ ResponseEntity<List<ExamOrderInfoResponse>> allExamOrderInfo(
       @RequestParam(name = "size", required = false, defaultValue = "10")
           @Min(value = 1, message = "invalid size")
           Integer size);
+
+  @Operation(
+      summary = "Retrieves all exam order records",
+      description =
+          "This endpoint does the same as exam-order-info, but it also slacks the list of exam orders.")
```

Comment:
Also unless it changed the other endpoint will return processed orders too and we don't want to slack those so maybe this needs to be more specific in the description?